Kconfig

```diff
@@ -335,6 +335,20 @@ menu "LVGL configuration"
 				which usually improves performance,
 				but does not guarantee the same rendering quality as the software.
 
+		config LV_VG_LITE_LINEAER_GRAD_CACHE_SIZE
```

Comment:
Let's use the name of `CNT` instead of `SIZE`, similar with header cache.

Reply:
Done.